samples/react-native/package.json

```diff
@@ -28,24 +28,25 @@
     "@sentry/react-native": "6.1.0",
     "delay": "^6.0.0",
     "react": "18.3.1",
-    "react-native": "0.75.4",
-    "react-native-gesture-handler": "^2.18.1",
-    "react-native-reanimated": "3.15.0",
-    "react-native-safe-area-context": "4.10.5",
-    "react-native-screens": "3.34.0",
-    "react-native-svg": "^15.6.0",
+    "react-native": "0.76.0",
+    "react-native-gesture-handler": "^2.20.2",
+    "react-native-reanimated": "3.16.1",
+    "react-native-safe-area-context": "github:th3rdwave/react-native-safe-area-context#main",
+    "react-native-screens": "4.0.0-beta.14",
+    "react-native-svg": "^15.8.0",
     "react-native-vector-icons": "^10.0.3",
     "react-redux": "^8.1.3",
     "redux": "^4.2.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.22.5",
-    "@babel/preset-env": "^7.22.5",
-    "@babel/runtime": "^7.22.5",
-    "@react-native/babel-preset": "0.75.4",
-    "@react-native/eslint-config": "0.75.4",
-    "@react-native/metro-config": "0.75.4",
-    "@react-native/typescript-config": "0.75.4",
+    "@babel/core": "^7.25.2",
+    "@babel/preset-env": "^7.25.3",
+    "@babel/runtime": "^7.25.0",
+    "@react-native-community/cli": "latest",
```

Comment:
[This is needed as a dependency now](https://reactnative.dev/blog/2024/10/23/release-0.76-new-architecture#removed-the-dependency-on-the-react-native-communitycli).

Reply:
The RN template specifies CLI 15, why we use the latest tag?

Reply:
Good catch @krystofwoldrich 🙇 
I've set it with 45e2fafa0c2b0d1cba273f7f40c49746dfbd6f17